src/app/views/collections/details/CollectionDetailsController.jsx

```diff
@@ -305,7 +306,27 @@ export class CollectionDetailsController extends Component {
         }
     };
 
-    handleCollectionPageClick = uri => {
+    handleCollectionPageClick = async (page, state) => {
+        const uri = page.uri;
+        if (this.props.isNewSignIn) {
+            const updateEmailInActiveCollection = {
+                ...this.props.activeCollection,
+            };
+            await user.getUserEmail(page.lastEdit.email).then(response => {
```

Comment:
Is it worth considering what we do if this request fails/errors? And might be worth considering what we show the user while we wait

Reply:
We'd probably want to add a log event as well as console log. e.g.`log.event("error getting user email address", error.log(error))` so that we send the error server side and is captured in Kibana.

Locally it will be fairly quick. Over a slow network and the VPN (which our users have to use) it might not be so quick. It's worth simulating slow network when testing such things. Dev tools/Inspect > Network > Throttling (slow 3g).

As with most things in newer parts of Florence i'd suggest we have a loading state, a loaded state and an error state. Something like:

Loading:
`Last edit: loading...`

or
`Last edit: loading... at (Mon 22 Jan 2024 - 15:20:17)`

Loaded: 
`Last edit: user@user.com (Mon 22 Jan 2024 - 15:20:17)`

Failed:
`Last edit: {UUID} at(Mon 22 Jan 2024 - 15:20:17)`

or even something to say it's failed. I don't think we need anything crazy and would be against designing something over the top. 

Happy for it to be run past UR first so it can be a new ticket. A fix the bug fix the bug ticket ;)